src/Application/Console/Commands/AnalyseCommand.php

```diff
@@ -55,6 +57,11 @@ public function __invoke(InputInterface $input, OutputInterface $output): int
 
         $formatter = FormatResolver::resolve($input, $output, $consoleOutput);
 
+        // flush cache before processing if asked
+        if ($input->getOption('flush-cache') === true) {
```

Comment:
You can use `hasOption` her instead?

Reply:
No, `hasOption` check if there is option in definition, not if option is setted 😕